samples/propgrid/propgrid.cpp

```diff
@@ -1930,6 +1865,70 @@ void FormMain::CreateGrid( int style, int extraStyle )
 
     InitPanel();
 
+    wxArrayString _fs_windowstyle_labels;
```

Comment:
And while I'm fine with moving these arrays into this function from global scope (although it's not that important...), I'd still keep them as arrays as well.

Reply:
Is wxString array (in place) initialization OK?  That requires C++11, will that be an issue? That's the main reason why I do push_back().

Reply:
OK, I changed these back to arrays.  Let me know if I shouldn't be initializing the wxString array in place like that.

Reply:
You can use `static const wxString array[] = { ... }` in C++98 without any problems. You can't initialize `wxArrayString` using initializer list like this (not even in C++11 as this class doesn't have any ctors taking `std::initializer_list`...).